templates/travel/partials/hero/hero.snip.html

```diff
@@ -142,8 +142,8 @@ <h2 class="travel-hero-subheading line-height-2 bold xs-hide sm-hide">Find and b
           Find Adventures & Tours
         </a>
       <!--/ Search Form -->
-      <a class="travel-hero-discover block center mx-auto mt1 md-hide lg-hide">
-        Discover adventures {{> ../../../../img/travel/icon/caret-down-small.svg}}
+      <a class="travel-hero-discover block center mx-auto mt1 md-hide lg-hide" on="tap:travel-landing-content.scrollTo">
```

Comment:
still keep an `href="#travel-landing-content"` in addition to the scrollTo to support "Open in new window" and copy link cases.

Reply:
Thanks ! Good catch!